config/externalname.go

```diff
@@ -1170,6 +1170,21 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	// Remediation Configurations can be imported using the name config_rule_name
 	"aws_config_remediation_configuration": config.ParameterAsIdentifier("config_rule_name"),
 
+	// appsync
+	//
+	// aws_appsync_api_cache can be imported using the AppSync API ID
+	"aws_appsync_api_cache": config.IdentifierFromProvider,
+	// aws_appsync_api_key can be imported using the AppSync API ID and key separated by :
+	"aws_appsync_api_key": config.IdentifierFromProvider,
```

Comment:
```suggestion
	"aws_appsync_api_key": TemplatedStringAsIdentifierWithNoName("{{ .parameters.api_id }}:{{ .external_name }}"),
```

Reply:
I have changed the externalname, thank you.